### PR TITLE
chore(browser): Session replay updates

### PR DESCRIPTION
**Pull request by ghost**
Post-release updates to the session replay doc (outstanding issues that couldn't make it into the first release)

### CODE REVIEW

**Comment by ally-sassman** [comment]
@bradleycamacho I'm aware of the one screenshot still in light mode, but I can't update it because of some issues getting a  test environment spun up, so I plan on updating all the images again once we go GA.

File: src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 title: "Session replay"
```
Comment (rhetoric101) [inline src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx:2]: @ally-sassman this is outside the scope of your current changes, but I'm not seeing a lot of "FAQ" headings in our docs. At some point, you might make merge that into "How session replay works" and just introduce them as questions about how the feature affects their environment.

**Review by rhetoric101 — DISMISSED** [review]
@ally-sassman I inserted a few comments you could check out. Also, in the step "Configure privacy settings" you might consider making h3 or h4 headings for mask, block, and ignore.

File: src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx (hunk 1 above)
Comment (ally-sassman) [reply src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx:2]: I posted something in the doc-talk about trying out a mini faq section for this LP doc, so I'll wait to see what Shawn and Austin think before removing/modifying it.

**Pull request merged**